cinema/views.py

```diff
@@ -167,6 +273,46 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type=str,
+                description="Filter date of movie session",
+                required=False,
+            ),
+            OpenApiParameter(
+                "movie",
+                type=int,
+                description="Filter by movie id",
+                required=False,
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        """Retrieve list of movies."""
+        return super().list(request, *args, **kwargs)
+
+    def create(self, request, *args, **kwargs):
+        """Created a new movie session."""
+        return super().create(request, *args, **kwargs)
+
+    def retrieve(self, request, *args, **kwargs):
+        """Retrieve detail movie session."""
+        return super().retrieve(request, *args, **kwargs)
+
+    def update(self, request, *args, **kwargs):
+        """Update full information about movie session."""
+        super().update(request, *args, **kwargs)
```

Comment:
The `update` method in `MovieSessionViewSet` is missing a `return` statement. Add `return` before `super().update(request, *args, **kwargs)` to ensure the response is returned correctly.